docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
     environment:
       DATABASE_URL: "postgres://postgres:chevas@db:5432/booksdb?sslmode=disable"
       DATABASE_MIGRATIONS_PATH: "/src/migrations"
+      DEADLINE: "10"
```

Comment:
Don't forget to change it here.
You can see why I am asking you to change the env variable name by looking at this env declaration in the docker-compose file here. What deadline stands for? You cannot tell looking from the outside. You would have to inspect the code to make sure what it actually represents.